awx/api/views/__init__.py

```diff
@@ -316,6 +316,9 @@ def get(self, request, format=None):
         if period == 'month':
             end_date = start_date - dateutil.relativedelta.relativedelta(months=1)
             interval = 'days'
+        elif period == 'two_weeks':
```

Comment:
@ryanpetrello this look good to you?  Just want to make sure we're not sneaking something by y'all without a review.